### PR TITLE
bump pocketpy with fixes

### PR DESCRIPTION
**Pull request by diamant3**

### CODE REVIEW

File: pocketpy/PSPBUILD [hunk 1]
```diff
@@ -1,24 +1,25 @@
 pkgname=pocketpy
-pkgver=1.0.9
-pkgrel=1
+pkgver=1.4.5
+pkgrel=2
```
Comment (sharkwouter) [inline pocketpy/PSPBUILD:3]: Since the version changed, the pkgrel should be 1.

**Review by sharkwouter — CHANGES_REQUESTED** [review]
Thanks, this looks pretty good. I had one suggestion.

File: pocketpy/PSPBUILD (hunk 1 above)
Comment (diamant3) [reply pocketpy/PSPBUILD]: Done. I forgot about that, thanks!

**Comment by diamant3** [comment]
This is irrelevant but i tried to fix the error in CI checks and my fix for this is should be the sources are from the github sdl repo instead of sdl website

error in CI:

```
curl: (60) SSL certificate problem: unable to get local issuer certificate
More details here: https://curl.se/docs/sslcerts.html

curl failed to verify the legitimacy of the server and therefore could not
establish a secure connection to it. To learn more about this situation and
how to fix it, please visit the web page mentioned above.
==> ERROR: Failure while downloading https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.6.3.tar.gz
    Aborting...
Error: Process completed with exit code 1.
```

can i make a PR for that?

**Pull request merged**